README.md

```diff
@@ -10,3 +10,16 @@ embedded in UOR artifacts.
 
 To learn more about Universal Runtime visit the [UOR-Framework](https://github.com/uor-framework/uor-framework) repository.
 
+## Basic Usage
+
+### Template content in a directory without pushing 
+```
+# The default workspace is "client-workspace" in the current working directory
+client build <directory> --output my-workspace
```

Comment:
I think this Usage section is a great idea, but it's too hard to understand - maybe we can add a longer flow, starting from how to build the client from source, how to start a local registry, and how to run the client a few times with different options. I think the initial audience of the README are the first contributors to the project, so we want as many people who see this first time to be able to get this working locally so they can hack with it a bit or try to integrate it into another client.

Reply:
I agree this README could jazzed up quite a bit to attract more people and the Usage section kind of comes out of nowhere. Great comment. I will tackle this on a follow-on PR to give it some thought.